tests/pr_test.go

```diff
@@ -66,3 +68,36 @@ func TestRunUpgradeExample(t *testing.T) {
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
```

Comment:
I suggest changing the upgrade test (`TestRunUpgradeExample`) to point to the DA code